doc/integrator/reset_password.rst

```diff
@@ -29,6 +29,11 @@ And to generate the required e-mail, in the ``vars_<package>.yaml`` file, add th
 
             Sincerely yours
             The GeoMapfish team
-        smtp_server: smtp.example.com
+        smtp:
+          host: smtp.example.com
+          ssl: true
+          user: <username>
+          password: <password>
+          starttls: false
 
```

Comment:
If the hostname ends with a colon (`:') followed by a number, and
        there is no port specified, that suffix will be stripped off and the
        number interpreted as the port number to use.

Reply:
you mean I need to document this?